pkg/querier/correctness/simple.go

```diff
@@ -5,11 +5,13 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/cortexproject/cortex/pkg/util/spanlogger"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/api/prometheus/v1"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
Why is this duplicated?

Reply:
I wondered about that too. It actually caused build failure. Fixing and running test locally as I write this.